R/pkg/R/context.R

```diff
@@ -435,3 +435,31 @@ setCheckpointDir <- function(directory) {
   sc <- getSparkContext()
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(directory))))
 }
+
+#' Total number of CPU cores of all executors registered in the cluster at the moment.
+#'
+#' @rdname spark.numCores
+#' @return current number of cores in the cluster.
+#' @examples
+#'\dontrun{
+#' spark.numCores()
+#'}
+#' @note spark.numCores since 2.4.0
+spark.numCores <- function() {
```

Comment:
still need to add these in https://github.com/MaxGekk/spark-1/blob/a39695e059c1a2976be50159e33144ee453d3c2f/R/pkg/NAMESPACE#L433